noxfile.py

```diff
@@ -93,7 +93,9 @@ def cover(session):
 @nox.session(python="3.9")
 def lint(session):
     """Run pre-commit linting."""
-    session.install("pre-commit")
+    # Pin virtualenv for pre-commit run
+    # See https://github.com/theacodes/nox/issues/545
+    session.install("virtualenv==20.10.0", "pre-commit")
```

Comment:
I don't think this was the right choice. Note virtualenv will use the latest setuptools in 28 days even on 20.10.0 :thinking:

Reply:
This might work for another 21 days... but then will break again.

Reply:
Any suggestions to deal with this?

Reply:
https://github.com/pre-commit/pre-commit/issues/2178#issuecomment-1002163763 forcing setuptools to use the stdlib via an env-var is likely a better approach

Reply:
I see. Maybe later I can post a PR for this.

Reply:
Thanks @gaborbernat 👍🏻

Reply:
This will be fixed in #549, thanks everyone